app/services/host_aspects/configuration.rb

```diff
@@ -0,0 +1,43 @@
+module HostAspects
+  class Configuration
+    SUBJECTS = [:configuration]
+
+    SUBJECTS.each do |subject|
+      define_method "register_#{subject}" do |aspect_name, aspect_model = nil, &block|
+        register(subject, aspect_name, aspect_model, &block)
+      end
+    end
+
+    SUBJECTS.each do |subject|
+      define_method "#{subject}_aspect" do
+        @aspects_registry[@aspect_subjects[subject]]
+      end
+    end
+
+    def registered_aspects
+      @aspects_registry.try(:dup) || {}
```

Comment:
Nitpick: this is a bit too clever. Maybe have an accessor `aspects_registry` that always returns an initialized instance var?